raiden/tests/unit/test_transfer.py

```diff
@@ -187,15 +187,23 @@ def test_mediated_transfer(raiden_network):
 
     amount = 10
 
-    alice_app.raiden.api.transfer(
+    result = alice_app.raiden.api.transfer_async(
         token_address,
         amount,
         charlie_address,
     )
 
-    gevent.sleep(1.)
+    assert channel_ab.locked == amount
+
+    # cannot do these assertions because the message has not been processed yet
+    # assert channel_ba.outstanding == amount
```

Comment:
Lack of synchronization points, don't think we should make a TODO, kept around to inform whomever is reading the code that this asserts were thought of.

Do you prefer to remove it?